pyproject.toml

```diff
@@ -74,6 +74,17 @@ azure-cosmos = {version="^4.4.0b1", optional=true}
 lark = {version="^1.1.5", optional=true}
 lancedb = {version = "^0.1", optional = true}
 pexpect = {version = "^4.8.0", optional = true}
+# Used for CI testing
```

Comment:
These won't be exported unless they're added here for some reason. Would be open to suggestions from any non-poetry novice